akka-persistence-typed/src/test/scala/akka/persistence/typed/serialization/ReplicatedEventSourcingSerializerSpec.scala

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (C) 2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.persistence.typed.serialization
+
+import akka.actor.testkit.typed.scaladsl.LogCapturing
+import akka.actor.testkit.typed.scaladsl.ScalaTestWithActorTestKit
+import akka.actor.testkit.typed.scaladsl.SerializationTestKit
+import akka.persistence.typed.PersistenceId
+import akka.persistence.typed.ReplicaId
+import akka.persistence.typed.crdt.Counter
+import akka.persistence.typed.crdt.ORSet
+import akka.persistence.typed.internal.PublishedEventImpl
+import akka.persistence.typed.internal.ReplicatedPublishedEventMetaData
+import akka.persistence.typed.internal.VersionVector
+import org.scalatest.wordspec.AnyWordSpecLike
+
+class ReplicatedEventSourcingSerializerSpec extends ScalaTestWithActorTestKit with AnyWordSpecLike with LogCapturing {
+
+  val testkit = new SerializationTestKit(system)
```

Comment:
rename since there is also `val testKit: ActorTestKit` from super class

Reply:
Doh there's one in the testkit